back-end/src/models/AppUser/AppUser.repository.ts

```diff
@@ -211,4 +287,27 @@ export default class AppUserRepository extends AppUserDb {
 
     return session.user;
   }
+
+  static async sendResetPasswordEmail(email: string): Promise<void> {
+    // Check if email exists in database
+    const user = await this.getUserByEmailAddress(email);
+    if (!user) {
+      throw new Error("Aucun utilisateur ne correspond à cet email.");
+    }
```

Comment:
same here. Already checked in getUserByEmailAddress() method

Reply:
ok

Reply:
Fix after VD codereview for the backend is ok